doc/man/cgget.1

```diff
@@ -27,6 +27,12 @@ This parameter can be used multiple times.
 .B -a, --all
 print the variables for all controllers which consist in the  given cgroup
 
+.TP
+.B -b
+Ignores the default systemd delegated hierarchy path and
```

Comment:
super nitpick - the other options _didn't_ capitalize the first letter.  Should we do the same here?